GSoC25/EntityLinking/test_redis.py

```
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+
+import redis
+import pandas as pd
+import time
+from typing import List, Dict
+
+
+class RedisEntityLinking:
+    
+    def __init__(self, host='91.99.92.217', port=6379, password='NEF!gsoc2025'):
```

Ruff (0.14.2)

Possible hardcoded password assigned to function default: "password"

(S107)